friends/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+ReactDOM.render(
+    <Router>
+        <App />
+    </Router>, document.getElementById('root'));
```

Comment:
Try to give arguments room the breathe when you've already split them up onto different lines:
```javascript
ReactDOM.render(
    <Router>
        <App />
    </Router>, 
    document.getElementById('root')
);
```